src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Build/IImplicitlyTriggeredBuildState.cs

```diff
@@ -15,6 +16,21 @@ namespace Microsoft.VisualStudio.ProjectSystem.Managed.Build
     [ProjectSystemContract(ProjectSystemContractScope.Global, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
     internal interface IImplicitlyTriggeredBuildState
     {
+        /// <summary>
+        /// Indicates if the current build (if any) is implicitly triggered.
+        /// </summary>
         bool IsImplicitlyTriggeredBuild { get; }
```

Comment:
Should this be `bool?` where `null` indicates no active build?

Reply:
We're not tracking builds in general, just the implicit ones; so we can't really distinguish explicit build/implicit build/no build.